src/components/content/monitor/MonitorChart.tsx

```diff
@@ -3,129 +3,184 @@
+    };
+
+    const handleChange = (value: string) => {
+        setColCountKey(Number(value));
+    };
+    const cols: JSX.Element[] = [];
+    let colCount = 0;
+    colColumnList.forEach((item) => {
+        if (item === Number(colCountKey)) {
+            colCount = item;
+        }
     });
 
+    for (let i = 0; i < options.length; i++) {
+        cols.push(
+            <Col key={i.toString()} span={24 / colCount}>
+                <div className={'charts-position-class'}>
+                    <div className={'chart-title-class'}>
+                        {chartsTitle[i].metricTitle}
+                        <br />
+                    </div>
+                    <div className={'chart-subtitle-class'}>
+                        <Tooltip
+                            placement='bottom'
+                            title={chartsTitle[i].Id}
+                            color={'#8ca2e4'}
+                            overlayStyle={{ maxWidth: 300 }}
+                        >
+                            <Button className={'chart-subtitle-hover-class'}>{chartsTitle[i].metricSubTitle}</Button>
+                        </Tooltip>
+                    </div>
+                    <EChartsReact option={options[i]} />
+                </div>
+            </Col>
+        );
+    }
+
+    const layOutOptions: { value: string; label: string }[] = getLayOutOptions();
+
     return (
         <>
-            {options.length > 0 ? (
-                <Row gutter={[16, 24]}>
-                    {options.map((option, index) => {
-                        return (
-                            <Col key={index} span={8}>
-                                <div className={'charts-position-class'}>
-                                    <EChartsReact option={option} />
-                                </div>
-                            </Col>
-                        );
-                    })}
-                </Row>
-            ) : (
-                <></>
-            )}
+            <div className={'charts-subtitle-option-class'}>
+                {' '}
+                <div className={'charts-radio-position-class'}>
+                    <Radio.Group onChange={onChangeTimePeriod} value={timePeriod}>
+                        {timePeriodList.map((item, index) => {
+                            return (
+                                <Radio key={index} value={index + 1}>
+                                    {item}
+                                </Radio>
+                            );
+                        })}
+                    </Radio.Group>
+                </div>
+                <Divider type='vertical' className={'charts-divider-class'} />
+                <div className={'charts-auto-refresh-class'}>
+                    Auto Refresh <Switch defaultChecked={false} onChange={onChangeAutoRefresh} />
+                </div>
+                <div className={'charts-column-count-position-class'}>
+                    Column Count:{' '}
```

Comment:
did not understand the use of this field. We can remove it i think.

Reply:
Keep this one.